tests/api2/test_auth_otp.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from middlewared.test.integration.utils import call, session, ssh, url
+
+
+@pytest.fixture(scope="module")
+def otp_enabled():
+    call("auth.twofactor.update", {"enabled": True})
+
+    try:
+        yield
+    finally:
+        ssh("midclt call auth.twofactor.update '{\"enabled\": false}'")
+
+
+def test_otp_http_basic_auth(otp_enabled):
```

Comment:
We need to test the specific scenario that was highlighted in the ticket. (i.e. using root credentials, try to disable 2fa globally over REST)